src/ui/SummaryDropdown/SummaryDropdown.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import SummaryDropdownComponent from './index'
+
+const meta: Meta<typeof SummaryDropdownComponent> = {
+  title: 'Components/SummaryDropdown',
+  component: SummaryDropdownComponent,
+}
+
+export default meta
+
+type Story = StoryObj<typeof SummaryDropdownComponent>
+
+export const SummaryDropdownMultiple: Story = {
+  render: () => (
+    <>
+      <SummaryDropdownComponent type="multiple">
+        <SummaryDropdownComponent.Item value="first-summary">
+          <SummaryDropdownComponent.Trigger>
+            First Summary
```

Comment:
JC will using `SummaryComponent.Title` change any styling?

Reply:
Hrrm ... it currently only bolds the text ... now re-thinking it ... I may just remove it straight up and just let people style everything themselves ...